jobs/felek.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+option go_package = "github.com/romanlevin/felek/jobs";
+
+package jobs;
+
+service Jobs {
+    rpc Start (JobStartRequest) returns (JobStatus);
+    rpc Stop (JobID) returns (JobStatus);
+    rpc Status (JobID) returns (JobStatus);
+    rpc Stdout (LogsRequest) returns (stream LogLine);
+    rpc Stderr(LogsRequest) returns (stream LogLine);
+}
+
+message JobStartRequest {
+    string path = 1;  // Path to executable
+    repeated string args = 2; // Represents argv, passed to the executable
+}
+
+message LogsRequest {
+    JobID id = 1; // ID of the job whose logs we want to get
+    bool follow = 2; // Keep streaming the logs as they come in
+    bool tail = 3; // If `follow=true`, don't stream existing lines before following
+}
+
+message JobStatus {
+    JobID id = 1; // A unique ID
+    oneof jobState {
+        RunningJob runningJob = 2;
+        StoppedJob stoppedJob = 3;
+    }
+}
+
+message RunningJob {
+    // XXX: Don't know if we only want this available for running processes,
```

Comment:
The PID might be useful in some cases but it's still fairly niche. It's fine to leave the message empty as a status indicator too.